match/match_test.go

```diff
@@ -97,47 +108,61 @@ func (mc *mockComm) CheckUser(userID int64) (bool, error) {
 	return false, nil
 }
 
-func makeRequest(env env, method string, url string, body string) (*httptest.ResponseRecorder, error) {
+func makeRequest(env env, method string, url string, body string, authToken string) (*httptest.ResponseRecorder, error) {
 	rec := httptest.NewRecorder()
 	req, err := http.NewRequest(method, url, strings.NewReader(body))
+	req.Header.Set("Authorization", "Bearer "+authToken)
 	if err != nil {
 		return nil, err
 	}
 	env.router().ServeHTTP(rec, req)
 	return rec, nil
 }
 
-// Test that a match list can be read successfully
+// Test that a match list can be read successfully for a given ID
 func TestListMatchHandlerSucceeds(t *testing.T) {
-	mockEnv := env{
-		&mockDAO{matchList: make([]dao.Match, 0)},
-		&mockComm{userIDs: []int64{0, 1, 2, 3}},
+	// Populate mock datastore
+	matchList := []dao.Match{
```

Comment:
Could you modify the list after making the mockEnv? Should mean we can one-line the creation of the empty env

Reply:
Yeah I thought the same, but cannot access the list - discussed offline